.release-please-manifest.json

```diff
@@ -1,3 +1,5 @@
 {
-  ".": "1.1.0"
+  "packages/server": "1.1.0",
+  "packages/client": "0.0.1-experimental",
+  "packages/shared": "0.0.1"
```

Comment:
The `shared` module is never published, but it's listed here so release-please can do it's magic and bump dependent packages when there's changes here.